<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="RaghavendraS" ignored="0" name="EmployeesRestAPITest" tests="34" failures="17" timestamp="05 Oct 2020 08:14:23 GMT" time="25.821" errors="0">
  <testcase name="checkContentLenght" time="0.004" classname="com.employeeapi.testCases.TC001_Get_All_Employees">
    <failure type="java.lang.NumberFormatException" message="null">
      <![CDATA[java.lang.NumberFormatException: null
at java.lang.Integer.parseInt(Unknown Source)
at java.lang.Integer.parseInt(Unknown Source)
at com.employeeapi.testCases.TC001_Get_All_Employees.checkContentLenght(TC001_Get_All_Employees.java:137)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- checkContentLenght -->
  <testcase name="checkContentType" time="0.002" classname="com.employeeapi.testCases.TC001_Get_All_Employees"/>
  <testcase name="checkCookies" time="0.001" classname="com.employeeapi.testCases.TC001_Get_All_Employees"/>
  <testcase name="checkResponseTime" time="0.001" classname="com.employeeapi.testCases.TC001_Get_All_Employees"/>
  <testcase name="checkResposeBody" time="0.032" classname="com.employeeapi.testCases.TC001_Get_All_Employees"/>
  <testcase name="checkStatusCode" time="0.002" classname="com.employeeapi.testCases.TC001_Get_All_Employees">
    <failure type="java.lang.AssertionError" message="expected [200] but found [429]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [429]
at com.employeeapi.testCases.TC001_Get_All_Employees.checkStatusCode(TC001_Get_All_Employees.java:63)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkStatusCode -->
  <testcase name="checkcontentEncoding" time="0.001" classname="com.employeeapi.testCases.TC001_Get_All_Employees"/>
  <testcase name="checkserverType" time="0.001" classname="com.employeeapi.testCases.TC001_Get_All_Employees">
    <failure type="java.lang.AssertionError" message="expected [nginx/1.14.1] but found [nginx/1.16.0]">
      <![CDATA[java.lang.AssertionError: expected [nginx/1.14.1] but found [nginx/1.16.0]
at com.employeeapi.testCases.TC001_Get_All_Employees.checkserverType(TC001_Get_All_Employees.java:113)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkserverType -->
  <testcase name="checkstatusLine" time="0.001" classname="com.employeeapi.testCases.TC001_Get_All_Employees">
    <failure type="java.lang.AssertionError" message="expected [HTTP/1.1 200 OK] but found [HTTP/1.1 429 Too Many Requests]">
      <![CDATA[java.lang.AssertionError: expected [HTTP/1.1 200 OK] but found [HTTP/1.1 429 Too Many Requests]
at com.employeeapi.testCases.TC001_Get_All_Employees.checkstatusLine(TC001_Get_All_Employees.java:90)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkstatusLine -->
  <testcase name="checkContentLenght" time="0.002" classname="com.employeeapi.testCases.TC002_Get_Single_Employee_Record"/>
  <testcase name="checkContentType" time="0.001" classname="com.employeeapi.testCases.TC002_Get_Single_Employee_Record">
    <failure type="java.lang.AssertionError" message="expected [text/html; charset=UTF-8] but found [application/json]">
      <![CDATA[java.lang.AssertionError: expected [text/html; charset=UTF-8] but found [application/json]
at com.employeeapi.testCases.TC002_Get_Single_Employee_Record.checkContentType(TC002_Get_Single_Employee_Record.java:79)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkContentType -->
  <testcase name="checkResponseTime" time="0.001" classname="com.employeeapi.testCases.TC002_Get_Single_Employee_Record"/>
  <testcase name="checkResposeBody" time="0.001" classname="com.employeeapi.testCases.TC002_Get_Single_Employee_Record">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.employeeapi.testCases.TC002_Get_Single_Employee_Record.checkResposeBody(TC002_Get_Single_Employee_Record.java:49)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkResposeBody -->
  <testcase name="checkStatusCode" time="0.0" classname="com.employeeapi.testCases.TC002_Get_Single_Employee_Record"/>
  <testcase name="checkserverType" time="0.0" classname="com.employeeapi.testCases.TC002_Get_Single_Employee_Record">
    <failure type="java.lang.AssertionError" message="expected [nginx/1.14.1] but found [nginx/1.16.0]">
      <![CDATA[java.lang.AssertionError: expected [nginx/1.14.1] but found [nginx/1.16.0]
at com.employeeapi.testCases.TC002_Get_Single_Employee_Record.checkserverType(TC002_Get_Single_Employee_Record.java:86)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkserverType -->
  <testcase name="checkstatusLine" time="0.001" classname="com.employeeapi.testCases.TC002_Get_Single_Employee_Record"/>
  <testcase name="checkContentType" time="0.002" classname="com.employeeapi.testCases.TC003_Post_Employee_Record"/>
  <testcase name="checkResposeBody" time="0.0" classname="com.employeeapi.testCases.TC003_Post_Employee_Record">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.employeeapi.testCases.TC003_Post_Employee_Record.checkResposeBody(TC003_Post_Employee_Record.java:71)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkResposeBody -->
  <testcase name="checkStatusCode" time="0.0" classname="com.employeeapi.testCases.TC003_Post_Employee_Record">
    <failure type="java.lang.AssertionError" message="expected [200] but found [429]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [429]
at com.employeeapi.testCases.TC003_Post_Employee_Record.checkStatusCode(TC003_Post_Employee_Record.java:81)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkStatusCode -->
  <testcase name="checkcontentEncoding" time="0.001" classname="com.employeeapi.testCases.TC003_Post_Employee_Record"/>
  <testcase name="checkserverType" time="0.0" classname="com.employeeapi.testCases.TC003_Post_Employee_Record"/>
  <testcase name="checkstatusLine" time="0.0" classname="com.employeeapi.testCases.TC003_Post_Employee_Record">
    <failure type="java.lang.AssertionError" message="expected [HTTP/1.1 200 OK] but found [HTTP/1.1 429 Too Many Requests]">
      <![CDATA[java.lang.AssertionError: expected [HTTP/1.1 200 OK] but found [HTTP/1.1 429 Too Many Requests]
at com.employeeapi.testCases.TC003_Post_Employee_Record.checkstatusLine(TC003_Post_Employee_Record.java:88)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkstatusLine -->
  <testcase name="checkContentType" time="0.002" classname="com.employeeapi.testCases.TC004_Put_Employee_Record">
    <failure type="java.lang.AssertionError" message="expected [text/html; charset=UTF-8] but found [application/json]">
      <![CDATA[java.lang.AssertionError: expected [text/html; charset=UTF-8] but found [application/json]
at com.employeeapi.testCases.TC004_Put_Employee_Record.checkContentType(TC004_Put_Employee_Record.java:95)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkContentType -->
  <testcase name="checkResposeBody" time="0.001" classname="com.employeeapi.testCases.TC004_Put_Employee_Record"/>
  <testcase name="checkStatusCode" time="0.001" classname="com.employeeapi.testCases.TC004_Put_Employee_Record"/>
  <testcase name="checkcontentEncoding" time="0.001" classname="com.employeeapi.testCases.TC004_Put_Employee_Record">
    <failure type="java.lang.AssertionError" message="expected [gzip] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [gzip] but found [null]
at com.employeeapi.testCases.TC004_Put_Employee_Record.checkcontentEncoding(TC004_Put_Employee_Record.java:109)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkcontentEncoding -->
  <testcase name="checkserverType" time="0.0" classname="com.employeeapi.testCases.TC004_Put_Employee_Record">
    <failure type="java.lang.AssertionError" message="expected [nginx/1.14.1] but found [nginx/1.16.0]">
      <![CDATA[java.lang.AssertionError: expected [nginx/1.14.1] but found [nginx/1.16.0]
at com.employeeapi.testCases.TC004_Put_Employee_Record.checkserverType(TC004_Put_Employee_Record.java:102)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkserverType -->
  <testcase name="checkstatusLine" time="0.001" classname="com.employeeapi.testCases.TC004_Put_Employee_Record"/>
  <testcase name="checkContentType" time="0.002" classname="com.employeeapi.testCases.TC005_Delete_Employee_Record">
    <failure type="java.lang.AssertionError" message="expected [text/html; charset=UTF-8] but found [application/json]">
      <![CDATA[java.lang.AssertionError: expected [text/html; charset=UTF-8] but found [application/json]
at com.employeeapi.testCases.TC005_Delete_Employee_Record.checkContentType(TC005_Delete_Employee_Record.java:82)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkContentType -->
  <testcase name="checkResposeBody" time="0.001" classname="com.employeeapi.testCases.TC005_Delete_Employee_Record">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.employeeapi.testCases.TC005_Delete_Employee_Record.checkResposeBody(TC005_Delete_Employee_Record.java:59)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkResposeBody -->
  <testcase name="checkStatusCode" time="0.0" classname="com.employeeapi.testCases.TC005_Delete_Employee_Record"/>
  <testcase name="checkcontentEncoding" time="0.001" classname="com.employeeapi.testCases.TC005_Delete_Employee_Record">
    <failure type="java.lang.AssertionError" message="expected [gzip] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [gzip] but found [null]
at com.employeeapi.testCases.TC005_Delete_Employee_Record.checkcontentEncoding(TC005_Delete_Employee_Record.java:96)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkcontentEncoding -->
  <testcase name="checkserverType" time="0.0" classname="com.employeeapi.testCases.TC005_Delete_Employee_Record">
    <failure type="java.lang.AssertionError" message="expected [nginx/1.14.1] but found [nginx/1.16.0]">
      <![CDATA[java.lang.AssertionError: expected [nginx/1.14.1] but found [nginx/1.16.0]
at com.employeeapi.testCases.TC005_Delete_Employee_Record.checkserverType(TC005_Delete_Employee_Record.java:89)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- checkserverType -->
  <testcase name="checkstatusLine" time="0.0" classname="com.employeeapi.testCases.TC005_Delete_Employee_Record"/>
</testsuite> <!-- EmployeesRestAPITest -->
